README.md

```diff
@@ -1,20 +1,67 @@
-# TSB contracts
+<p align="center"><a href="https://sandbox.game"><img width="300" title="TSB" src='brand-assets/tsb_3d.jpeg' /></a></p>
 
-This workspace uses [Nx, a Smart, fast and extensible build system.](https://nx.dev) 
+<h1 align="center">The Sandbox smart contracts</h1>
 
-## Understand this workspace
+![Node Version](https://img.shields.io/badge/node-18.x-green)
+[![Discord](https://img.shields.io/discord/497312527093334036.svg?label=Discord&logo=discord)](<https://discord.gg/vAe4zvY>)
 
+[The Sandbox](https://sandbox.game) is a user-generated content (UGC) gaming platform, that will empower creators through digital ownership and monetization of 3D voxel creations made and shared by users around the world.
+
+This mono-repo contains The Sandbox smart contracts, underpinning The Sandbox metaverse. Our key token contracts were conceived inside the core package. You can find out more about some of our early blockchain features in [this article](https://medium.com/sandbox-game/blockchain-features-in-the-sandbox-7db91fcc615c). Since core was created, we have added new contracts and updates via new packages - see the below Architecture Overview for more information.
+
+## Learn more
+
+- [Website](www.sandbox.game)
+- [Discord](https://discordapp.com/invite/vAe4zvY)
+- [Telegram](https://t.me/sandboxgame)
+- [Medium](https://medium.com/sandbox-game)
+
+## Prerequisites
+
+- ⚙️ [NodeJS](https://nodejs.org/)
+- 🧰 [Yarn](https://yarnpkg.com/)
```

Comment:
specify the version of yarn / node needed

Reply:
in badge above

Reply:
although I'm not clear on the minimum requirement